oval-schemas/unix-definitions-schema.xsd

```diff
@@ -2252,6 +2252,111 @@
+                  <xsd:documentation>
+                        The sshd_test is used to check the values associated with sshd parameters that are used by the local system. It extends the 
+                        standard TestType as defined in the oval-definitions-schema and one should refer to the TestType description for more information. 
+                        The required object element references a sshd_object and the optional state element references a sshd_state that specifies the 
+                        information to check.
+                  </xsd:documentation>
+                  <xsd:appinfo>
+                        <oval:element_mapping>
+                              <oval:test>sshd_test</oval:test>
+                              <oval:object>sshd_object</oval:object>
+                              <oval:state>sshd_state</oval:state>
+                              <oval:item target_namespace="http://oval.mitre.org/XMLSchema/oval-system-characteristics-5#unix">sshd_item</oval:item>
+                        </oval:element_mapping>
+                  </xsd:appinfo>
+                  <xsd:appinfo>
+                        <sch:pattern id="unix-def_sshdtst">
+                              <sch:rule context="unix-def:sshd_test/unix-def:object">
+                                    <sch:assert test="@object_ref=ancestor::oval-def:oval_definitions/oval-def:objects/unix-def:sshd_object/@id"><sch:value-of select="../@id"/> - the object child element of a sshd_test must reference a sshd_object</sch:assert>
+                              </sch:rule>
+                              <sch:rule context="unix-def:sshd_test/unix-def:state">
+                                    <sch:assert test="@state_ref=ancestor::oval-def:oval_definitions/oval-def:states/unix-def:sshd_state/@id"><sch:value-of select="../@id"/> - the state child element of a sshd_test must reference a sshd_state</sch:assert>
+                              </sch:rule>
+                        </sch:pattern>
+                  </xsd:appinfo>
+            </xsd:annotation>
+            <xsd:complexType>
+                  <xsd:complexContent>
+                        <xsd:extension base="oval-def:TestType">
+                              <xsd:sequence>
+                                    <xsd:element name="object" type="oval-def:ObjectRefType" />
+                                    <xsd:element name="state" type="oval-def:StateRefType" minOccurs="0" maxOccurs="unbounded"/>
+                              </xsd:sequence>
+                        </xsd:extension>
+                  </xsd:complexContent>
+            </xsd:complexType>
+      </xsd:element>
+      <xsd:element name="sshd_object" substitutionGroup="oval-def:object">
+            <xsd:annotation>
+                  <xsd:documentation>
+                        The sshd_object is used by a sshd_test to define which sshd parameters on the local system should be collected via the "sshd -T [NAME]" command. 
```

Comment:
The `sshd -T [NAME]` is probably incorrect - the OpenSSH client doesn't expect any arguments if supplied with the `-T` option, and it just dumps configuration keys.